### DISCUSSION TITLE
trouble installing O3DE

### DISCUSSION THREAD

**Discussion by Amanda-Dearheart** [o3de/o3de#6186] [Help Wanted]
Im having trouble installing O3DE.  I've attached a copy of the error log file for anyone with the time and expertise to discover the problems I am having.

[O3DE_[Stable]_20211206174742.log](https://github.com/o3de/o3de/files/7664909/O3DE_.Stable._20211206174742.log)

**Comment by amznestebanpapp** [comment]
According to the log, it was not able to download the installer. The current installer is an "online installer", it will download the different package files.
Are you connected to the internet while executing the installer?

It could also been related to the AWS outage from some days ago, the package files are stored in S3. 

Can you try again?
